buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -96,9 +103,62 @@ class PrecommitTasks {
         }
         Task forbiddenApis = project.tasks.findByName('forbiddenApis')
         forbiddenApis.group = "" // clear group, so this does not show up under verification tasks
+
         return forbiddenApis
     }
 
+    private static Task configureForbiddenApisCli(Project project) {
+        project.configurations.create("forbiddenApisCliJar")
+        project.dependencies {
+            forbiddenApisCliJar 'de.thetaphi:forbiddenapis:2.5'
+        }
+        Task forbiddenApisCli = project.tasks.create('forbiddenApisCli')
+        project.ext.getForbiddenSignaturesFile = { name -> project.getForbiddenSignaturesFile(name) }
```

Comment:
Do we need this one? It looks circular.

Reply:
It's so we can call `getForbiddenSignaturesFile(name)` from a build script instead of `import ... ; PrecommitTasks.getForbiddenSignaturesFile(project, name)`

Reply:
I'll remove it for now, and add it back once it's actually used.